package.json

```diff
@@ -6,23 +6,17 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "yext pages build && yext pages generate-test-data && pages dev",
-    "build:local": "pages build",
-    "build": "yext pages build && yext pages generate-test-data && yext pages render",
-    "build:serve": "npm run build && yext pages serve",
-    "serve": "yext pages serve",
-    "features": "pages generate features",
-    "fmt": "prettier -w .",
-    "lint": "eslint .",
-    "clean": "rm -rf .artifact-output .yext/ dist/ localData sites-rendered-output",
-    "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite"
+    "dev": "pages dev",
+    "prod": "pages prod",
+    "build": "pages build"
   },
   "engines": {
-    "node": ">=17"
+    "node": "^18.0.0 || >=20.0.0"
   },
   "dependencies": {
     "@headlessui/react": "^1.7.12",
     "@heroicons/react": "^2.0.14",
+    "@yext/sites-components": "1.0.0-rc.5",
```

Comment:
Should be pages-components. Add caret to everything. Update all other deps besides react. Use pagesjs rc.7.